scheduler/scheduler-server/src/test/java/performancetests/verification/VerifyChangeTest.java

```diff
@@ -111,14 +113,15 @@ public void changeShouldBeLessThanThreshold() throws IOException {
     }
 
     private List<File> getTwoLastFiles() {
-        final String pathToStorage = System.getProperty("pathToStorage");
-        File folder = new File(pathToStorage);
+        File folder = new File(System.getProperty("pa.rm.home"));
```

Comment:
So the performance files will be at the root of `pa.rm.home`?

Reply:
Yes. I use Copy to Slave plugin of Jenkins. And it copies files directly to the workspace of slave == pa.rm.home